docs/UserGuide.md

```diff
@@ -101,9 +101,19 @@ Format: `list -t today`
   * `-m`m: Morning (Midnight to 12 Noon)
```

Comment:
You could remove the extra 'm' at line 101.

Reply:
Fixed in #b59de3d